networking_aci/plugins/ml2/drivers/mech_aci/agent/aci_agent.py

```diff
@@ -147,13 +147,6 @@ def _check_and_handle_signal(self):
             LOG.info(_LI("Agent caught SIGTERM, quitting daemon loop."))
             self.run_daemon_loop = False
             self.catch_sigterm = False
-        if self.catch_sighup:
```

Comment:
Would it make sense to also remove the `self.catch_sighup` attribute?

Reply:
Yes